src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtImpPrebid.java

```diff
@@ -29,4 +29,15 @@
      */
     @JsonProperty("storedbidresponse")
     List<ExtStoredBidResponse> storedBidResponse;
+
+    /**
+     * Defines the contract for bidrequest.imp[i].ext.prebid.is_rewarded_inventory
+     */
+    Boolean isRewardedInventory;
+
+    public static ExtImpPrebid of(ExtStoredRequest storedrequest,
```

Comment:
This seems like a hack. What if you'll need to add another field?
Please, use `@Builder` instead of `@AllArgsConstructor`.

PS. Usually, we use builder for models having more then 3 fields.